design/LoadBalancing/LoadBalancing.md

```diff
@@ -0,0 +1,221 @@
+# Load Balancing in FoundationDB
+
+In FoundationDB, often multiple *interface*s are available for the same type of *request*s. A load balancer can be used to distribute the requests to those interfaces, while awaring the possible failures.
+
+Two load balancer are provided: `basicLoadBalance` and `loadBalance`, both defined in `LoadBalance.actor.h`. The `basicLoadBalance` is a simple 
```

Comment:
Looks like this sentence is not finished?

Reply:
Thanks for the catch! I have rephrased the whole section.